generate_tactile_switches.py

```diff
@@ -600,109 +789,241 @@ def _uuid(identifier: str) -> str:
-                Attribute('FORCE', '130gf', AttributeType.STRING, None),
-            ]),
-            Part('PTS645SM952LFS', Manufacturer('C&K'), [
-                Attribute('FORCE', '160gf', AttributeType.STRING, None),
-            ]),
-            Part('PTS645SN952LFS', Manufacturer('C&K'), [
-                Attribute('FORCE', '100gf', AttributeType.STRING, None),
-            ]),
-            Part('PTS645SH952LFS', Manufacturer('C&K'), [
-                Attribute('FORCE', '200gf', AttributeType.STRING, None),
-            ]),
-            Part('PTS645SP952LFS', Manufacturer('C&K'), [
-                Attribute('FORCE', '320gf', AttributeType.STRING, None),
-            ]),
-            Part('PTS645SR952LFS', Manufacturer('C&K'), [
-                Attribute('FORCE', '360gf', AttributeType.STRING, None),
-            ]),
-        ], common_part_attributes=[
-            Attribute('HEIGHT', '9.5mm', AttributeType.STRING, None),
-        ]),
+        Model(
+            name='PTS645Sx952',
+            actuator_height=9.5,
+            parts=[
+                Part(
+                    'PTS645SK952LFS',
+                    Manufacturer('C&K'),
+                    [
+                        Attribute('FORCE', '260gf', AttributeType.STRING, None),
+                    ],
+                ),
+                Part(
+                    'PTS645SL952LFS',
+                    Manufacturer('C&K'),
+                    [
+                        Attribute('FORCE', '130gf', AttributeType.STRING, None),
+                    ],
+                ),
+                Part(
+                    'PTS645SM952LFS',
+                    Manufacturer('C&K'),
+                    [
+                        Attribute('FORCE', '160gf', AttributeType.STRING, None),
+                    ],
+                ),
+                Part(
+                    'PTS645SN952LFS',
+                    Manufacturer('C&K'),
+                    [
+                        Attribute('FORCE', '100gf', AttributeType.STRING, None),
+                    ],
+                ),
+                Part(
+                    'PTS645SH952LFS',
+                    Manufacturer('C&K'),
+                    [
+                        Attribute('FORCE', '200gf', AttributeType.STRING, None),
+                    ],
+                ),
+                Part(
+                    'PTS645SP952LFS',
+                    Manufacturer('C&K'),
+                    [
+                        Attribute('FORCE', '320gf', AttributeType.STRING, None),
+                    ],
+                ),
+                Part(
+                    'PTS645SR952LFS',
+                    Manufacturer('C&K'),
+                    [
+                        Attribute('FORCE', '360gf', AttributeType.STRING, None),
+                    ],
+                ),
+            ],
+            common_part_attributes=[
+                Attribute('HEIGHT', '9.5mm', AttributeType.STRING, None),
+            ],
+        ),
```

Comment:
Hmm not sure about this.... it doesn't look bad, but it is *way* more verbose than before, making it harder to read IMO